docs/self_hosting/configuration/blob_storage.mdx

```diff
@@ -82,7 +83,9 @@ and set the blob storage [container](https://learn.microsoft.com/en-us/azure/sto
 
 1. [Storage account and access key](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage)
 1. [Connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string)
-1. Environment variables, Workload identity, or managed identity supported by [`DefaultAzureCredential`](https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure). This is the default authentication method when configuration for either option above is not present.
+1. Environment variables, [Workload identity](https://azure.github.io/azure-workload-identity/docs/introduction.html), or managed identity supported by [`DefaultAzureCredential`](https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure). This is the default authentication method when configuration for either option above is not present.
```

Comment:
could we mark this as the recommended (for both aws/azure)

Reply:
done!